x/payments/handler.go

```diff
@@ -96,7 +96,19 @@ func handleMsgSetPaymentContractAuthorisation(ctx sdk.Context, k Keeper, msg Msg
 		return err.Result()
 	}
 
-	return sdk.Result{}
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypePaymentContractAuthorisation,
+			sdk.NewAttribute(types.AttributeKeyPayerDid, msg.PayerDid),
+			sdk.NewAttribute(types.AttributeKeyPaymentContractId, msg.PaymentContractId),
```

Comment:
Here we are missing the `msg.Authorised` value as an attribute.

Reply:
Ok. Added the msg.Authorised